Src/PChecker/CheckerCore/SystematicTesting/OperationScheduler.cs

```diff
@@ -178,7 +178,7 @@ internal void ScheduleNextEnabledOperation(AsyncOperationType type)
             }
 
             ScheduledOperation = next as AsyncOperation;
-            ScheduleTrace.AddSchedulingChoice(next.Id);
+            ScheduleTrace.AddSchedulingChoice(next.Name);
```

Comment:
I dont understand this change?

Reply:
This makes the *.schedule file much more friendly to understand while debugging (for us at least as we develop different checkers). This change makes printing a scheduling choice (as well as reading while replaying) as the actor/machine name, instead of printing an id.

Reply:
There is no performance or any side effects due to this change.

Reply:
Reverted via https://github.com/p-org/P/pull/614/commits/60506120fcd3a6e626511a8934e530f5550b0a87